bundles/org.openhab.binding.liquidcheck/src/main/java/org/openhab/binding/liquidcheck/internal/httpclient/LiquidCheckHttpClient.java

```diff
@@ -0,0 +1,119 @@
+import org.openhab.binding.liquidcheck.internal.LiquidCheckConfiguration;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * The {@link LiquidCheckHttpClient} sets up the jetty client for the connection to the device.
+ *
+ * @author Marcel Goerentz - Initial contribution
+ */
+@NonNullByDefault
+public class LiquidCheckHttpClient {
+
+    private final Logger logger = LoggerFactory.getLogger(LiquidCheckHttpClient.class);
+    private final HttpClient client;
+    private final LiquidCheckConfiguration config;
+
+    public boolean isClosed = false;
+
+    /**
+     * The Constructor of the LiquidCheckHttpClient class will set up a jetty client
+     * 
+     * @param config
+     */
+    public LiquidCheckHttpClient(LiquidCheckConfiguration config) {
+        this.config = config;
+        client = new HttpClient();
+
+        try {
+            client.setFollowRedirects(false);
+            client.setName("LiquidCheckHttpClient");
+            client.setIdleTimeout(config.connecionTimeout * 1000);
+            client.start();
+        } catch (Exception e) {
+            logger.error("Couldn't start Client! Exception: {}", e.toString());
+            return;
+        }
+    }
+
+    /**
+     * The pollData method will poll the data from device
+     * 
+     * @return String with the response of the request
+     * @throws InterruptedException
+     * @throws TimeoutException
+     * @throws ExecutionException
+     */
+    public String pollData() throws InterruptedException, TimeoutException, ExecutionException {
+        String uri = "http://" + config.hostname + "/infos.json";
+        Request request = client.newRequest(uri);
+        request.method(HttpMethod.GET);
+        ContentResponse response = request.send();
+        return response.getContentAsString();
+    }
+
+    /**
+     * The measureCommand method will start a measurement
+     * 
+     * @return String with response of the request
+     * @throws InterruptedException
+     * @throws TimeoutException
+     * @throws ExecutionException
+     */
+    public String measureCommand() throws InterruptedException, TimeoutException, ExecutionException {
+        String uri = "http://" + config.hostname + "/command";
+        Request request = client.newRequest(uri);
+        request.method(HttpMethod.POST);
+        request.header(HttpHeader.CONTENT_TYPE, "applicaton/json");
+        request.content(new StringContentProvider(
+                "{\"header\":{\"namespace\":\"Device.Control\",\"name\":\"StartMeasure\",\"messageId\":\"1\",\"payloadVersion\":\"1\"},\"payload\":null}"));
+        ContentResponse response = request.send();
+        return response.getContentAsString();
+    }
+
+    /**
+     * The isConnected method will return the state of the http client
+     * 
+     * @return
+     */
+    public boolean isConnected() {
+        String state = this.client.getState();
+        return "STARTED".equals(state) ? true : false;
```

Comment:
Just `return "STARTED".equals(state);`

Reply:
Note that you do not need this method if using the common HTTP client.
The common HTTP client will be started when your thing handler will initialized.